app/aboutdialog.cpp

```diff
@@ -106,8 +107,8 @@ SOFTWARE.
 )"));
 
     QStringList thirdPartyLibsStr {
-        QStringLiteral("<h1 align='center'><b>%1</b></h1>").arg(tr("Third-party Libraries used by %1")),
-        tr("%1 is built on the following free software libraries:"),
+        QStringLiteral("<h1 align='center'><b>%1</b></h1>").arg(tr("Third-party libraries used by %1")),

```

Comment:
Not sure if this should be title case. Is it a headline?

Reply:
![image](https://user-images.githubusercontent.com/10095765/102083336-563ad700-3e4e-11eb-889d-40702d5a3439.png)
Yes, the formats are the same between all other tabs.